commands/cmdutils/cmdutils.go

```diff
@@ -163,15 +163,21 @@ func LabelsPrompt(response *[]string, apiClient *gitlab.Client, repoRemote *glre
 		if err != nil {
 			return err
 		}
-		*response = selectedLabels
+		// Add each element to the array instead of just assigning it so we respect values already
+		// present, given from the `--label` flag
+		for _, x := range selectedLabels {
+			*response = append(*response, x)
+		}
```

Comment:
You should simplify this:
```suggestion
		*response = append(*response, selectedLabels...)
```

Reply:
Yes I saw the deepsource, never crossed my mind to use the `...` thing

Reply:
Sorry, this was resolved in the subsequent commits